bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -89,12 +92,13 @@ public static String executeUrl(String httpMethod, String url, int timeout) thro
      * Furthermore the <code>http.proxyXXX</code> System variables are read and
      * set into the {@link HttpClient}.
      *
-     * @param httpMethod the HTTP method to use
-     * @param url the url to execute
-     * @param content the content to be send to the given <code>url</code> or <code>null</code> if no content should be
-     *            send.
+     * @param httpMethod  the HTTP method to use
+     * @param url         the url to execute
+     * @param content     the content to be send to the given <code>url</code> or <code>null</code> if no content should
+     *                        be
```

Comment:
Could you please fix all those line breaks throughout the PR?

Reply:
Will do, they have been done by the default formatting Eclipse Save Action after installing ESH Eclipse via Oomph btw, so that should be changed in the installer imho?

Reply:
The formatter keeps previous line breaks, but it won't add two with one word each line by itself. I therefore do not think that the formatter settings need to be changed.